copilot/fsd-pre-award/manifest.yml

```diff
@@ -12,91 +12,97 @@ http:
   # To match all requests you can use the "/" path.
   path: "/"
   # You can specify a custom health check path. The default is "/".
-  healthcheck: "/healthcheck"
+  healthcheck:
+    path: "/healthcheck"
+    healthy_threshold: 2
+    interval: 6s
+    timeout: 5s
+    grace_period: 20s
 
   alias:
     - frontend.${COPILOT_ENVIRONMENT_NAME}.access-funding.test.levellingup.gov.uk
     - assessment.${COPILOT_ENVIRONMENT_NAME}.access-funding.test.levellingup.gov.uk
     - authenticator.${COPILOT_ENVIRONMENT_NAME}.access-funding.test.levellingup.gov.uk
+# Your service is reachable at "http://fsd-pre-award.${COPILOT_SERVICE_DISCOVERY_ENDPOINT}:8080" but is not public.
 
 # Configuration for your containers and service.
 image:
-  # Docker build arguments. For additional overrides: https://aws.github.io/copilot-cli/docs/manifest/lb-web-service/#image-location
-  location: ghcr.io/communitiesuk/funding-service-pre-award-frontend:latest
   # Port exposed through your container to route traffic to it.
   port: 8080
 
-# Valid values: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html
-# Number of CPU units for the task.
-cpu: 1024
-# Amount of memory in MiB used by the task.
-memory: 2048
-
-# See https://aws.github.io/copilot-cli/docs/manifest/lb-web-service/#platform
-platform: linux/x86_64
-# Number of tasks that should be running in your service.
-count: 2
-# Enable running commands in your container.
-exec: true
+cpu: 2048 # Number of CPU units for the task.
+memory: 4096 # Amount of memory in MiB used by the task.
+platform: linux/x86_64 # See https://aws.github.io/copilot-cli/docs/manifest/backend-service/#platform
+count: 2 # Number of tasks that should be running in your service.
+exec: true # Enable running commands in your container.
 
 network:
   connect: true # Enable Service Connect for intra-environment traffic between services.
+  vpc:
+    security_groups:
+      - from_cfn: ${COPILOT_APPLICATION_NAME}-${COPILOT_ENVIRONMENT_NAME}-fsdpreawardstoresclusterSecurityGroup
 
 # storage:
 # readonly_fs: true       # Limit to read-only access to mounted root filesystems.
 
 # Optional fields for more advanced use-cases.
-#
 # Pass environment variables as key value pairs.
 variables:
+  FLASK_ENV: ${COPILOT_ENVIRONMENT_NAME}
+#  SENTRY_DSN: "https://80c7f65b54f0eff535777a66b375adf0@o1432034.ingest.us.sentry.io/4508324370317312"  # FIXME: UPDATE THIS WITH NEW PROJECT?
```

Comment:
Do we want to follow up on this? It's some effort to set up projects/ line up with repos so seems sensible this would be done last. Just thinking through implication on not seeing failures through during deploy

Reply:
will add this back 👍